agent/functional_tests/tests/functionaltests_windows_test.go

```diff
@@ -36,8 +41,12 @@ const (
 	savedStateTaskDefinition        = "savedstate-windows"
 	portResContentionTaskDefinition = "port-80-windows"
 	labelsTaskDefinition            = "labels-windows"
+	dockerEndpoint                  = "npipe:////./pipe/docker_engine"
+	DockerEndpointEnvVariable       = "DOCKER_HOST"
 )
 
+var endpoint = utils.DefaultIfBlank(os.Getenv(DockerEndpointEnvVariable), dockerEndpoint)
```

Comment:
If this is only used by GMSA, why this needs to be added outside the function itself? Also, **DockerEndpointEnvVariable** does not need be an exported env var.

Reply:
done.